libc/socket/unix-socket.c

```diff
@@ -859,7 +860,7 @@ void unix_poll(int type)
 	TEST_ASSERT(rv == 0);
 	TEST_ASSERT(fds[0].revents == 0);
 	TEST_ASSERT(fds[1].revents == 0);
-	TEST_ASSERT_LESS_THAN(350, ms);
+	TEST_ASSERT_LESS_THAN(300 + pollTimeoutDelay, ms);
```

Comment:
This causes the bound on `ms` to decrease by 20ms in the default case (`pollTimeoutDelay = 30`). Is this intended?

Reply:
As I remember it was set 310 initialy but due to fails on emulated target it was increased to 350 and it worked out that it is still too low for zynq-qemu, so in this pr zynq-qemu gets its own custom value, and all others target will use default value of 30 ms, I think it will be sufficient.

Reply:
Alright, thanks